### ISSUE TITLE
Image includes do not seem to be included in non-default branches

### ISSUE THREAD

**Issue by mkobit** [christiangalsterer/bitbucket-asciidoc-plugin#12]
1. I have a `master` as my default branch
2. I push up some `.adoc` with some `image::[]` blocks and some `.png`s to `my-branch`
3. I view the new rendered page

the images do not seem to be included in the rendering of `my-branch`.
It looks like they are only included when merged into the default branch.

**Comment by christiangalsterer** [comment]
Which plugin and Bitbucket version do you use?

**Comment by christiangalsterer** [comment]
I have tried it with the latest version of the plugin and the latest version of Bitbucket and I can't reproduce the issue. Can you doublecheck that all files have been pushed/merged correctly?

**Comment by mkobit** [comment]
@christiangalsterer sorry, I haven't look back at this but will check again today.

**Comment by mkobit** [comment]
Sorry for taking so long to get back to you, I don't think I described the issue well so hopefully the steps below clarify the issue

Here are the steps I followed locally using the `atlas-run-standalone -p 7990 --context-path / --version 4.10.0 --product bitbucket` and testing with https://github.com/asciidoctor/jekyll-asciidoc.
1. Run `atlas-run-standalone -p 7990 --context-path / --version 4.10.0 --product bitbucket`
2. Login with `admin`/`admin`
3. Install the `2.3.1` version
4. Create a repository
5. Clone down https://github.com/asciidoctor/jekyll-asciidoc, change upstream
   
   ```
   git clone https://github.com/asciidoctor/jekyll-asciidoc.git
   cd jekyll-asciidoc
   git remote set-url origin http://admin@localhost:7990/scm/project_1/my-repo.git
   ```
6. Checkout out new branch
   
   ```
   git checkout -b issue-12-repro
   ```
7. Download an image
   
   ```
   curl https://www.brandeps.com/logo-download/B/Bitbucket-01.png -o bitbucket-logo.png
   ```
8. Make a tweak to the `README.adoc` to use `include::[]` macro.
   
   ```
   == Overview
   
   Here is the Bitbucket logo!
   
   image::bitbucket-logo.png[]
   ```
9. Add, commit, and push
   
   ```
   git add README.adoc bitbucket-logo.png
   ```
10. Preview a pull request to the `master` branch (http://localhost:7990/projects/PROJECT_1/repos/my-repo/compare/commits?sourceBranch=refs/heads/issue-12-repro)
    
    ![Pull request preview](https://cloud.githubusercontent.com/assets/1447148/20240663/a47f5a8a-a8e3-11e6-9929-9ac398d54a1e.png)
11. Select the _"full source"_ button
    
    ![View full source button](https://cloud.githubusercontent.com/assets/1447148/20240666/c3e9367a-a8e3-11e6-8a8f-4ce17f52e073.png)
12. See broken image on page 
    
    ![Broken image link and source](https://cloud.githubusercontent.com/assets/1447148/20240682/34c5460e-a8e4-11e6-8e31-8d92afedb4f6.png)

**Comment by christiangalsterer** [comment]
Hi @mkobit,

thank you very much for the detailed description. I was now able to reproduce the issue. The problem is that the link is created against the configured default branch.

I will look into it and will have a fix hopefully soon.

**Comment by christiangalsterer** [comment]
The fix is included in the new version 2.3.2 available in the Atlassian Marketplace.
